certified-connectors/DBF2XML/apiDefinition.swagger.json

```diff
@@ -8,10 +8,7 @@
   "info": {
     "description": "A service that allows content to be converted from dbf format to another.",
     "title": "DBF2XML",
-    "version": "1.0.0",
-    "x-ms-api-annotation": {
-      "status": "Preview"
-    }
+    "version": "1.0.0"
```

Comment:
Version string seems inconsistent. Is it 1.0.0, or 1.0, or v1?

Reply:
I have removed only  the "x-ms-api-annotation" properties from swagger files and I didn't touch not versions. Versions are showing in the PR, because I have removed the comma after the version property, but if you observe them, they are not changed in my PR